certified-connectors/DocuSignDemo/script.csx

```diff
@@ -950,6 +950,27 @@ public class Script : ScriptBase
     return body;
   }
 
+  private JObject AddRemindersBodyTransformation(JObject body)
+  {
+    var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
+    var newBody = new JObject()
+    {
+      ["useAccountDefaults"] = "false",
+      ["reminders"] = new JObject()
+      {
+        ["reminderDelay"] = query.Get("reminderDelay"),
+        ["reminderEnabled"] = query.Get("reminderEnabled"),
+        ["reminderFrequency"] = query.Get("reminderFrequency")
+      },
+      ["expirations"] = new JObject()
+      {
+        ["expireAfter"] = "120"
```

Comment:
can a number be assigned to this instead of string ?

Reply:
I still need to convert to string to pass to the API. So I made a string assignment.